src/sentry/utils/geo.py

```diff
@@ -19,14 +19,10 @@
     geoip_path = getattr(settings, 'GEOIP_PATH', None)
     if geoip_path:
         try:
-            geo_db = GeoIP.open(geoip_path, GeoIP.GEOIP_STANDARD)
-        except BaseException:
+            geo_db = GeoIP.open(geoip_path, GeoIP.GEOIP_MEMORY_CACHE)
+        except Exception:
             logger.warning("Error opening GeoIP database: %s" % geoip_path)
         else:
-            def geo_by_addr(ip):
-                try:
-                    return geo_db.record_by_addr(ip)
-                except BaseException:
-                    return None
+            geo_by_addr = lambda ip: geo_db.record_by_addr(ip)
```

Comment:
I think you been writing Java or something too long.

You don't even need the lambda here. Just `geo_by_addr = geo_db.record_by_addr`.

Reply:
Refactor fail, I've never programmed Java. :eye: